Docker_files/ocsci_container/Containerfile.ci

```diff
@@ -36,13 +36,21 @@ RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/s
     && mv kubectl /usr/local/bin/
 
 RUN curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar -C /usr/local/bin -zxvf - oc
+COPY /Docker_files/ocsci_container/centos.repo /etc/yum.repos.d/
+COPY /Docker_files/ocsci_container/google-chrome.repo /etc/yum.repos.d/
 
-RUN dnf install -y --nodocs python38 git jq rsync make \
+# Install EPEL repos
+# RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm https://dl.fedoraproject.org/pub/epel/epel-next-release-latest-8.noarch.rpm
```

Comment:
Is this line commented out intentionally?
If it's not needed, we can delete 2 lines above, right?

Reply:
Yes, but I left them as an option to enable them if in the future we will need packages from epel, for example pulling jq and yq instead of downloading directly from github.